src/api/api.js

```diff
@@ -0,0 +1,30 @@
+import { request } from './../helpers/fetchHelpers';
+
+export const login = body => {
+    return request('login', 'post', body)
+}
+
+export const getProjects = () => {
+    return request('projects', 'post', null)
+}
+
+export const addNewProjects = (body) => {
+    return request('addProjects', 'post', body)
+}
+
+export const removeProject = (body) => {
+    return request('removeProject', 'post', body)
+}
+
+export const updateProjectDescription = (body) => {
+    return request('editProjectDescription', 'post', body)
+}
+
+export const updateProjectBeginningDate = (body) => {
+    return request('editProjectBeginningDate', 'post', body)
+}
+
+export const updateProjectDueDate = (body) => {
+    console.log(body)
+    return request('editProjectDueDate', 'post', body)
+}
```

Comment:
- Please utilise naming a file to `index.js` if that file represents that directory. (/src/api/api.js -> /src/api/index.js)
- Remove all unnecessary console logs
- Some more basic linting issues: 
 - in case of single argument no need to add _Parentheses_ `()` around.